website_sale_product_subscribe_notify/views/res_config_settings_views.xml

```diff
@@ -0,0 +1,22 @@
+<?xml version="1.0" encoding="utf-8"?>
+<odoo>
+
+    <record id="res_config_settings_view_form" model="ir.ui.view">
+        <field name="name">res.config.settings.view.form</field>
+        <field name="model">res.config.settings</field>
+        <field name="inherit_id" ref="website.res_config_settings_view_form"/>
```

Comment:
@TimLai125 I put the limit_recipient setting in website's setting. Do you think it is good?

Reply:
I think the setting is not related to the website, there maybe General Settings is a better place for this.

Reply:
ok